### DISCUSSION TITLE
Remove @achrinza/node-ipc from the dependencies

### DISCUSSION THREAD

**Discussion by jokay** [vuejs/vue-cli#7397] [Q&A]
Hello

Would it be possible to remove the dependency to [@achrinza/node-ipc](https://github.com/achrinza/node-ipc) from [@vue/cli-shared-utils](https://www.npmjs.com/package/@vue/cli-shared-utils) somehow?

After every major Node.js update we cannot update because of this package, see [achrinza/node-ipc/#45](https://github.com/achrinza/node-ipc/issues/45).

**Comment by sinisarudan** [comment]
Adding ' `--ignore-engines`' parameter in most cases helps, yet I and many other developers would like to get rid of this unhealthy dependency too
